linkup-cli/src/commands/status.rs

```diff
@@ -40,6 +40,16 @@ pub fn status(args: &Args) -> anyhow::Result<()> {
         println!("{}", warning.yellow());
     }
 
+    if !LocalState::exists() {
+        println!(
+            "{}",
+            "\nSeems like you don't have any state yet, so there is no status to report.".yellow()
+        );
+        println!("{}", "Have you run `start` at least once?\n".yellow());
```

Comment:
```suggestion
        println!("{}", "Have you run `linkup start` at least once?\n".yellow());
```
Maybe it's obvious and not needed? 🤷

Reply:
Updated on [4c27287](https://github.com/mentimeter/linkup/pull/228/commits/4c2728709cdbbc28c2521fb720a576cc3f82b7eb)